src/librustc_mir/borrow_check/used_muts.rs

```diff
@@ -52,6 +52,24 @@ struct GatherUsedMutsVisitor<'visit, 'cx: 'visit, 'gcx: 'tcx, 'tcx: 'cx> {
     mbcx: &'visit mut MirBorrowckCtxt<'cx, 'gcx, 'tcx>,
 }
 
+impl<'visit, 'cx, 'gcx, 'tcx> GatherUsedMutsVisitor<'visit, 'cx, 'gcx, 'tcx> {
```

Comment:
```suggestion
impl GatherUsedMutsVisitor<'_, '_, '_, '_> {
```

Reply:
Shoot I didn't know we could do that.  That's great!